src/semantics/index.ts

```diff
@@ -205,8 +206,27 @@ function createFunctionDeclareNode(
     const this_type = getMethodClassType(f, context);
     const parameters: VarDeclareNode[] = [];
     if (f.genericOwner) {
-        f.genericOwner.paramArray.forEach((p) => {
-            f.addParameter(p);
+        const genericOwner = f.genericOwner as FunctionScope;
+        const specializedArgs = f.funcType.specializedArguments!;
+        genericOwner.paramArray.forEach((v) => {
+            const specializedType = TypeResolver.createSpecializedType(
+                v.varType,
+                specializedArgs,
+                (f.genericOwner as FunctionScope).funcType,
+            );
+            const newParam = new Parameter(
+                v.varName,
+                specializedType,
+                v.varModifiers,
+                v.varIndex,
+                v.isOptional,
+                v.destructuring,
+            );
+            if (v.initExpression) newParam.setInitExpr(v.initExpression);
+            newParam.setIsLocalVar(v.isLocalVar());
+            newParam.needReBinding = v.needReBinding;
+            newParam.tsNode = v.tsNode;
+            f.addParameter(newParam);
```

Comment:
So the specialized function scope is created in frontend, and the parameters are added in semantic tree, and it is same as class specialization too?

Reply:
Yes，the same goes for generic member functions in generic class.